nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/settings/ui/reporting-tasks/reporting-task-table/reporting-task-table.component.html

```diff
@@ -111,53 +111,64 @@
-                        @if (hasAdvancedUi(item)) {
-                            <div
-                                class="pointer fa fa-cogs primary-color"
-                                (click)="advancedClicked(item, $event)"
-                                title="Advanced"></div>
-                        }
-                        @if (canStop(item)) {
-                            <div
-                                class="pointer fa fa-stop primary-color"
-                                (click)="stopClicked(item)"
-                                title="Stop"></div>
-                        }
-                        @if (canStart(item)) {
-                            <div
-                                class="pointer fa fa-play primary-color"
-                                (click)="startClicked(item)"
-                                title="Start"></div>
-                        }
-                        @if (canChangeVersion(item)) {
-                            <div
-                                class="pointer fa fa-exchange primary-color"
-                                (click)="changeVersionClicked(item)"
-                                title="Change Version"></div>
-                        }
-                        @if (canDelete(item)) {
-                            <div
-                                class="pointer fa fa-trash primary-color"
-                                (click)="deleteClicked(item)"
-                                title="Delete"></div>
-                        }
-                        @if (canViewState(item)) {
-                            <div
-                                class="pointer fa fa-tasks primary-color"
-                                (click)="viewStateClicked(item)"
-                                title="View State"></div>
-                        }
-                        @if (canManageAccessPolicies()) {
-                            <div
-                                class="pointer fa fa-key primary-color"
-                                (click)="$event.stopPropagation()"
-                                [routerLink]="getPolicyLink(item)"
-                                title="Access Policies"></div>
-                        }
+                        <button
+                            mat-icon-button
+                            type="button"
+                            [matMenuTriggerFor]="actionMenu"
+                            class="h-16 w-16 flex items-center justify-center icon global-menu">
+                            <i class="fa fa-ellipsis-v"></i>
+                        </button>
+                        <mat-menu #actionMenu="matMenu" xPosition="before">
+                            <button mat-menu-item (click)="configureClicked(item, $event)">
+                                <i class="fa fa-cog primary-color mr-2"></i>
+                                {{ canEdit(item) ? 'Edit' : 'View Configuration' }}
```

Comment:
Edit opens the dialog but does not close the menu:
<img width="1058" alt="Screenshot 2024-05-07 at 5 11 35 PM" src="https://github.com/apache/nifi/assets/713866/ddcf1a4b-2725-4c87-a3b1-c104cecbd840">

Reply:
https://github.com/apache/nifi/pull/8766#discussion_r1594323472